CMakeLists.txt

```diff
@@ -98,6 +98,12 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/bin")
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/lib")
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/library")
 
+if(FLB_SYSTEM_WINDOWS)
```

Comment:
The conditional disable should be done in the [windows specific cmake file](https://github.com/fluent/fluent-bit/blob/master/cmake/windows-setup.cmake#L6)